src/lib.rs

```diff
@@ -180,13 +223,25 @@ fn inner(markdown: &str, options: Options, h1_level: u8) -> Result<Vec<u8>, Stri
                 }
             }
 
+            InlineMath(s) => {
+                // We use #inlinemath(`…`) for inline math
+                result.extend_from_slice(b"#inlinemath(`");
+                result.extend_from_slice(s.as_bytes());
```

Comment:
This should probably be escaped, no?

Reply:
I don't think so. Having escaped enabled will cause commands such as `\int` to be escaped as `\\int`, resulting in an error display.

Reply:
You would have to make a different escaping function – the point is just so that including backticks in the LaTeX doesn’t cause things to break.

Reply:
It's too difficult to include escaped backtick in raw, and we can consider using four backticks instead of one.

Reply:
Then how about using `#inlinemath(raw("` and a normally-escaped string?

Reply:
OK, I thought typst didn't support line breaks in str, but in fact it could. I replaced it with what you said.